tests/common/snappi_tests/traffic_generation.py

```diff
@@ -344,9 +344,10 @@ def run_traffic(duthost,
         cs.state = cs.START
         api.set_capture_state(cs)
 
-    clear_dut_interface_counters(duthost)
-
-    clear_dut_que_counters(duthost)
+    for duthost in [*snappi_extra_params.multi_dut_params.ingress_duthosts,
```

Comment:
The duthost value will be changed by this loop. can you include duthost in the list and use set to avoid duplication?

Reply:
Hi @sdszhang, the duthost will be included automatically as it's path of the egress_duthosts added previously. I've updated the variable duthost to host so that it doesn't overwrite

Reply:
sometime, ingress_dut maybe same with egress_dut.
can use use set() to avoid clearing the same dut twice?